redisinsight/ui/src/App.tsx

```diff
@@ -1,44 +1,56 @@
 import { hot } from 'react-hot-loader/root'
 import React, { ReactElement } from 'react'
-import { Provider } from 'react-redux'
+import { Provider, useSelector } from 'react-redux'
 import { EuiPage, EuiPageBody } from '@elastic/eui'
 
+import { appInfoSelector } from 'uiSrc/slices/app/info'
+import { PagePlaceholder } from 'uiSrc/components'
 import Router from './Router'
 import store from './slices/store'
 import { Theme } from './constants'
 import { themeService } from './services'
-import { NavigationMenu, Notifications, Config, ShortcutsFlyout, MonitorConfig } from './components'
+import { Config, MonitorConfig, NavigationMenu, Notifications, ShortcutsFlyout } from './components'
 import { ThemeProvider } from './contexts/themeContext'
 import MainComponent from './components/main/MainComponent'
 
 import themeDark from './styles/themes/dark_theme/_dark_theme.lazy.scss'
 import themeLight from './styles/themes/light_theme/_light_theme.lazy.scss'
 
 import './App.scss'
+import { BuildType } from "uiSrc/constants/env";
```

Comment:
move this import after other `uiSrc/` imports. Also, please, change to single quotes

Reply:
done